tests/st/libnetwork/test_mainline_single_host.py

```diff
@@ -43,14 +54,14 @@ def test_mainline(self):
 
             # Allow network to converge
             # Check connectivity.
-            workload1.assert_can_ping(workload2.ip, retries=5)
-            self.assert_connectivity([workload1, workload2])
+            workload1.assert_can_ping("workload2", retries=5)
+            workload2.assert_can_ping("workload1", retries=5)
 
             # Disconnect endpoints from the network
             # Assert can't ping and endpoints are removed from Calico
             network.disconnect(host, workload1)
             network.disconnect(host, workload2)
-            workload1.assert_cant_ping(workload2.ip, retries=5)
+            workload1.assert_cant_ping("workload2", retries=5)
```

Comment:
Not really a valid test since host names are not added for other networks.  I think for cross network, we still need to use IP.